build.gradle

```diff
@@ -37,6 +37,8 @@ sourceCompatibility = '1.6'
 
 dependencies {
   compile 'org.jenkins-ci.lib:dry-run-lib:0.1'
+  compile 'org.jenkins-ci:symbol-annotation:1.3'
+  jenkinsPlugins 'org.jenkins-ci.plugins:structs:1.3@jar'
```

Comment:
Uh, should suffice to add the `structs` dependency without specifying `symbol-annotation`, no?